doc/source/index.rst

```diff
@@ -2,161 +2,199 @@
+
+.. grid:: 2 2 3 4
+   :gutter: 3
+
+   .. grid-item-card:: Key Feature 1
+      :text-align: center
+      
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. 
+
+   .. grid-item-card::
+      :text-align: center
+
+      Key Feature 2 (Variant)
+      ^^^
+      
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. 
+
+   .. grid-item-card:: Key Feature 3
+      :text-align: center
+      
+      Description of Key Feature
+
+   .. grid-item-card:: Key Feature 4
+      :text-align: center
+      
+      Description of Key Feature
+
+   .. grid-item-card:: Key Feature 5
+      :text-align: center
+      
+      Description of Key Feature
+
+   .. grid-item-card:: Key Feature 6
+      :text-align: center
+      
+      Description of Key Feature
+
+   .. grid-item-card:: Key Feature 7
+      :text-align: center
+
+      Description of Key Feature
+
+   .. grid-item-card:: Key Feature 8
+      :text-align: center
+      
+      Description of Key Feature
+
+
+Support ArviZ
+=============
+
+.. raw:: html 
+
+   <div class="two-col">
+      <!-- Contributions -->
+      <div class="col" id="contributions">
+         <h3>Contributions</h3>
+         <p>Contributions and issue reports are very welcome at <a href="https://github.com/arviz-devs/arviz">the GitHub repository</a>. We have a <a href="https://github.com/arviz-devs/arviz/blob/main/CONTRIBUTING.md">contributing guide</a> to help you through the process. If you have any doubts, please do not hesitate to contact us on <a href="https://gitter.im/arviz-devs/community">gitter</a>.</p>
+      </div>
+      <!-- Citation -->
+      <div class="col" id="citation">
+         <h3>Citation</h3>
+         <p>If you use ArviZ and want to <strong>cite</strong> it please use <a class="reference external" href="https://doi.org/10.21105/joss.01143"><img alt="JOSS" src="https://joss.theoj.org/papers/10.21105/joss.01143/status.svg"></a>.</p>
+         <p>See our <a href="https://www.arviz.org/en/latest/support.html#cite">support page</a> for information on how to cite in BibTeX format.</p>
+      </div>
+   </div>
+
+   <div class="two-col">
+      <!-- Sponsors -->
+      <div class="col" id="sponsors">
```

Comment:
I might add sponsors, institutional partners and numfocus logo at the bottom with a centered "Sponsors and institutional partners" title and no explanation. 

Then we could do something like:
1. set the content to be wider and use a single 3 element grid for contributing, citing and donating.
2. Use the grid to mention/explain numfocus as we do now and then have 3 "buttons/calls-to-action": donate link like we have now, get ArviZ merchandise ([full logo](https://numfocus.myspreadshop.com/arviz+logo?idea=629e289fc8ee26344a684241) and [favicon logo](https://numfocus.myspreadshop.com/arviz+favicon+design?idea=62a74f17ebe60a221692c6f2) both available) and maybe a 3rd one about becoming a sponsor (we could even write/add a page to arviz.org with sponsor prospectus)

Reply:
> Is this layout what you have in mind? 

Option 1 would be restructuring the elements we have now only. We currently have 1 section with 4 squares:

```
## Support ArviZ
contributions                citation
sponsors                     donate
```

they would become

```
## Support ArviZ
contributions      citation      donate

## Sponsors and institutional partners
<-- logos only --!>
```

Example of sponsor section: https://xarray.dev/#sponsors

---

option 2 would be something similar to what we have now:

```
## Support ArviZ
contributions            citation
numfocus (description)   calls to action (donate, buy swag, sponsor)

## Sponsors ...
logos only
```

2x3 grid with more space per element and one button per grid item also sounds good, but all 6 squares would go inside support arviz, then the sponsors section would come.

Reply:
> Re: setting content to be wider, the current website and draft has all sections at the same width (see below), but if we have 3 elements per row for this grid then I will likely need to make this specific section wider than the others. I can give it a try to see what works.

The full width text that we currently have inside the key features section has to go, so I would try making everything a bit wider (not full width, maybe not even as wide as the navbar content, but a bit wider). I'd also expect that to be easier and less strange/confusing for readers